.github/workflows/packages.yml

```diff
@@ -12,13 +12,25 @@ jobs:
       - name: Build sdist
         run: python setup.py sdist
 
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+
+      - name: Available platforms
+        run: echo ${{ steps.qemu.outputs.platforms }}
```

Comment:
I assume this is a debug statement?

Reply:
docker/setup-qemu-action@v1 is required to run aarch64 docker container on x86 builders. However we can remove line 19 and 20 which is just listing what platforms qemu supports.